docs/docs/cmd/cli/cli-issue.md

```diff
@@ -0,0 +1,34 @@
+# cli issue
+
+Gets access token for the specified resource
```

Comment:
Let's grab the description from the command and put it in here 😄

Reply:
Whoops my copy paste skills are killing me here 🗡️